sdk/docs/KFServingApi.md

```diff
@@ -0,0 +1,137 @@
+# KFServingApi
+
+Class | Method |  Description
+------------ | ------------- | -------------
+KFServingApi | [deploy](#deploy) | Create the provided KFService in the specified namespace|
+KFServingApi | [get](#get)    | Get the deployed KFService in the specified namespace|
+KFServingApi | [patch](#patch)  | Patch the deployed KFService in the specified namespace |
+KFServingApi | [delete](#delete) | Delete the deployed KFService in the specified namespace |
+
+## deploy
+> deploy(kfservice, namespace='default')
```

Comment:
We have also namespace on kfservice object, it maybe confusing which namespace we are respecting.

Reply:
Updated to "namespace=None" for all APIs. 

For now the logic are as following if namespace is not set while calling the apis:

1. For `deploy` and `patch`, try to get the namespace from kfservice definition firstly, if did not get that (None), then will get default_namespace (If in k8s cluster, get the current namespace, if not, set the namespace to 'default' lastly).
2. For `get` and `delete`, since no kfservice definition, so just derectly use default_namespace (If in k8s cluster, get the current namespace, if not, set the namespace to 'default' lastly)

Updated example and sample accordingly, set the namespace in the kfservice definition for `deploy` and `patch`, but keep as before for `get` and `delete`.